releases/release-5.4.0.md

```diff
@@ -78,6 +78,7 @@ In v5.4, the key new features or improvements are as follows:
 - TiDB Server, PD Server, and TiKV Server start using a unified naming method for the log-related parameters to manage log names, output formats, and the rules for rotation and expiration. For details, see [TiKV configuration file - log](/tikv-configuration-file.md#log-new-in-v540).
 - Since v5.4.0, if you create a SQL binding for an execution plan that has been cached via Plan Cache, the binding invalidates the plan already cached for the corresponding query. The new binding does not affect execution plans cached before v5.4.0.
 - In v5.3 and earlier versions, [TiDB Data Migration (DM)](https://docs.pingcap.com/tidb-data-migration/v5.3/) documentation is independent of TiDB documentation. Since v5.4, DM documentation is integrated into TiDB documentation with the same version. You can directly read [DM documentation](/dm/dm-overview.md) without accessing the DM documentation site.
+- Remove the experimental feature of Point-in-time recovery (PITR) along with cdclog. Since v5.4.0, PITR and cdclog are no longer supported.
```

Comment:
The TiCDC / cdclog based PITR isn't supported, but I think binlog / reparo ( https://docs.pingcap.com/tidb/stable/tidb-binlog-reparo ) is still supported.

"Remove the experimental feature of Point-in-time recovery (PITR) along with cdclog. Since v5.4.0, cdclog based PITR and cdclog are removed."

Reply:
```suggestion
- Remove the experimental feature of Point-in-time recovery (PITR) along with cdclog. Since v5.4.0, cdclog-based PITR and cdclog are no longer supported.
```